autotest-agent/src/jvmMain/kotlin/com/epam/drill/agent/test/testinfo/TestInfoSender.kt

```diff
@@ -50,16 +50,18 @@ class IntervalTestInfoSender(
 
     override fun stopSendingTests() {
         scheduledThreadPool.shutdown()
-        if (!scheduledThreadPool.awaitTermination(5, TimeUnit.SECONDS)) {
+        if (!scheduledThreadPool.awaitTermination(1, TimeUnit.SECONDS)) {
```

Comment:
Any particular reason we want to send it each second instead of once per 5 seconds?

Reply:
We don't really need to wait 5 seconds since there is another scheduler with 5 seconds wait timer, its gona be fine